### PR TITLE
Remove oldColors

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #2704

### CODE REVIEW

File: packages/atlas/src/components/Carousel/Carousel.stories.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ const CarouselItem = styled.div`
   width: 300px;
   min-height: 200px;
   text-align: center;
-  color: ${oldColors.white};
-  background-color: ${oldColors.gray['300']};
+  color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/Carousel/Carousel.stories.tsx]: we have `colorCoreBaseWhite` and we should use that

File: packages/atlas/src/components/Tabs/Tabs.styles.ts [hunk 2]
```diff
@@ -40,14 +40,14 @@ export const Tab = styled.div<TabProps>`
   font: ${cVar('typographyDesktopT200')};
   letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
   text-transform: ${cVar('typographyDesktopT200TextTransform')};
-  color: ${({ selected }) => (selected ? oldColors.white : oldColors.gray[300])};
+  color: ${({ selected }) => (selected ? 'white' : cVar('colorCoreNeutral300'))};
```
Comment (drillprop) [inline packages/atlas/src/components/Tabs/Tabs.styles.ts]: we have colorCoreBaseWhite and we should use that

File: packages/atlas/src/components/_buttons/CallToActionButton/CallToActionButton.styles.ts [hunk 3]
```diff
@@ -54,7 +54,7 @@ export const StyledContainer = styled('button', { shouldForwardProp: isPropValid
   align-items: center;
   cursor: pointer;
   border: 0;
-  color: ${oldColors.white};
+  color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_buttons/CallToActionButton/CallToActionButton.styles.ts]: same as above

File: packages/atlas/src/components/_inputs/Checkbox/Checkbox.styles.ts [hunk 4]
```diff
@@ -64,15 +64,15 @@ export type CheckboxStateProps = {
 }
 export const InnerContainer = styled.div<CheckboxStateProps>`
   transition: all 0.125s ease;
-  color: ${oldColors.white};
-  border: 1px solid ${oldColors.gray[300]};
+  color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_inputs/Checkbox/Checkbox.styles.ts]: same as above

File: packages/atlas/src/components/_inputs/RadioInput/RadioInput.styles.ts [hunk 5]
```diff
@@ -134,7 +132,7 @@ export const CustomRadioInput = styled.div<CustomRadioInputProps>`
   ${colorFromProps};
 
   & + span {
-    color: ${(props) => (props.checked ? oldColors.white : '')};
+    color: ${(props) => (props.checked ? 'white' : '')};
```
Comment (drillprop) [inline packages/atlas/src/components/_inputs/RadioInput/RadioInput.styles.ts]: same as above

File: packages/atlas/src/components/_inputs/Select/Select.styles.ts [hunk 6]
```diff
@@ -98,8 +98,8 @@ export const SelectMenu = styled.ul<SelectMenuProps>`
   z-index: ${zIndex.globalOverlay};
   padding: 0;
   margin-top: ${({ isOpen }) => (isOpen ? sizes(1) : 0)};
-  background-color: ${oldColors.gray[700]};
-  color: ${oldColors.white};
+  background-color: ${cVar('colorCoreNeutral700')};
+  color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_inputs/Select/Select.styles.ts]: same as above

File: packages/atlas/src/components/_inputs/Slider/Slider.styles.ts [hunk 7]
```diff
@@ -11,35 +11,35 @@ export const StyledRcSlider = styled(RcSlider)`
...
-    background-color: ${oldColors.blue['500']};
+    background-color: ${cVar('colorCoreBlue500')};
   }
 
   .rc-slider-handle {
     width: ${THUMB_SIZE}px;
     height: ${THUMB_SIZE}px;
     margin-top: ${THUMB_SIZE / -2 + TRACK_HEIGHT / 2}px;
-    background-color: ${oldColors.white};
+    background-color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_inputs/Slider/Slider.styles.ts]: same as above

File: packages/atlas/src/components/_inputs/TitleArea/TitleArea.styles.ts [hunk 8]
```diff
@@ -19,9 +19,9 @@ export const TitleAreaInfo = styled.div<TitleAreaInfoProps>`
 `
 
 export const StyledTextArea = styled(TextareaAutosize)`
-  caret-color: ${oldColors.blue[500]};
+  caret-color: ${cVar('colorCoreBlue500')};
   color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_inputs/TitleArea/TitleArea.styles.ts]: same as above

File: packages/atlas/src/components/_navigation/BottomNav/BottomNav.styles.ts [hunk 9]
```diff
@@ -28,7 +28,7 @@ export const Container = styled.div`
 `
 
 export const NavLink = styled(Link, { shouldForwardProp: isPropValid })<{ active: PathMatch | null }>`
-  color: ${oldColors.white};
+  color: white;
```
Comment (drillprop) [inline packages/atlas/src/components/_navigation/BottomNav/BottomNav.styles.ts]: same as above

File: packages/atlas/src/styles/GlobalStyles.tsx [hunk 10]
```diff
@@ -58,7 +57,7 @@ const globalStyles = css`
   h5,
   h6 {
     font-family: ${cVar('typographyFontsPrimary')};
-    color: ${oldColors.white};
+    color: white;
```
Comment (drillprop) [inline packages/atlas/src/styles/GlobalStyles.tsx]: same as above

File: packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.styles.ts [hunk 11]
```diff
@@ -30,7 +30,7 @@ const topbarFocusStyles = ({ hasFocus }: FocusProps) =>
         }
       `
     : css`
-        background-color: ${oldColors.black};
+        background-color: black;
```
Comment (drillprop) [inline packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.styles.ts]: We have `colorCoreBaseBlack` and we should use that

File: packages/atlas/src/components/_video/VideoCategoryCard/FeaturedVideoCategoryCard.style.ts [hunk 12]
```diff
@@ -41,7 +41,7 @@ export const FeaturedIconCircle = styled(IconCircle)`
   margin-bottom: ${sizes(4)};
 
   path {
-    fill: ${oldColors.black};
+    fill: black;
```
Comment (drillprop) [inline packages/atlas/src/components/_video/VideoCategoryCard/FeaturedVideoCategoryCard.style.ts]: same as above

File: packages/atlas/src/styles/GlobalStyles.tsx [hunk 13]
```diff
@@ -47,8 +46,8 @@ const globalStyles = css`
 
   body {
     font-family: ${cVar('typographyFontsSecondary')};
-    background: ${oldColors.black};
-    color: ${oldColors.gray[50]};
+    background: black;
```
Comment (drillprop) [inline packages/atlas/src/styles/GlobalStyles.tsx]: same as above

File: packages/atlas/src/views/viewer/CategoryView/CategoryVideos.styles.ts [hunk 14]
```diff
@@ -32,8 +32,8 @@ export const ControlsContainer = styled.div`
 
   /* re-add global horizontal padding to the component */
   padding: ${sizes(4)} var(--size-global-horizontal-padding);
-  border-bottom: 1px solid ${oldColors.gray[700]};
-  background-color: ${oldColors.black};
+  border-bottom: 1px solid ${cVar('colorCoreNeutral700')};
+  background-color: black;
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/CategoryView/CategoryVideos.styles.ts]: same as above

File: packages/atlas/src/components/FiltersBar/FiltersBar.stories.tsx [hunk 15]
```diff
@@ -77,7 +77,7 @@ const ControlsContainer = styled.div`
   grid-template-columns: 1fr auto;
   align-items: center;
   padding-bottom: ${sizes(4)};
-  border-bottom: 1px solid ${oldColors.gray[700]};
+  border-bottom: 1px solid ${cVar('colorCoreNeutral700')};
   z-index: 100;
   position: relative;
   background-color: black;
```
Comment (drillprop) [inline packages/atlas/src/components/FiltersBar/FiltersBar.stories.tsx]: same as above

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_video/VideoHero/VideoHero.styles.ts [hunk 16]
```diff
@@ -41,9 +41,9 @@ export const GradientOverlay = styled.div`
   right: 0;
   bottom: 0;
   left: 0;
-  background: linear-gradient(180deg, transparent 50%, ${oldColors.black} 93.23%, ${oldColors.black} 100%),
-    radial-gradient(50.66% 101.32% at 50% 50%, transparent 0%, ${oldColors.transparentBlack[54]} 100%),
-    ${oldColors.transparentBlack[54]};
+  background: linear-gradient(180deg, transparent 50%, black 93.23%, black 100%),
```
Comment (kdembler) [inline packages/atlas/src/components/_video/VideoHero/VideoHero.styles.ts]: Should we use token for black here as well, for consistency?

**Review by kdembler — APPROVED** [review]

**Pull request merged**